x-pack/plugins/security/public/management/roles/edit_role/validate_role.test.ts

```diff
@@ -161,6 +161,74 @@ describe('validateIndexPrivileges', () => {
   });
 });
 
+describe('validateRemoteIndexPrivileges', () => {
+  beforeEach(() => {
+    validator = new RoleValidator({ shouldValidate: true });
+  });
+
+  test('it ignores empty remote privileges', () => {
+    const role = {
+      name: '',
+      elasticsearch: {
+        indices: [],
+        remote_indices: [
+          {
+            clusters: [],
+            names: [],
+            privileges: [],
+          },
+        ],
+        cluster: [],
+        run_as: [],
+      },
+      kibana: [],
+    };
+
+    expect(validator.validateRemoteIndexPrivileges(role)).toEqual({
+      isInvalid: false,
+    });
+  });
+
+  test('it requires privileges when an index is defined', () => {
```

Comment:
Nit: Could add a test for "It requires `indices` and `privileges` when `clusters` is defined"